parcels/particle.py

```diff
@@ -169,7 +168,7 @@ class ScipyParticle(_Particle):
     dt = Variable('dt', dtype=np.float32, to_write=False)
     state = Variable('state', dtype=np.int32, initial=ErrorCode.Success, to_write=False)
 
-    def __init__(self, lon, lat, fieldset, depth=0., dt=1., time=0., cptr=None):
+    def __init__(self, lon, lat, fieldset, depth=0., time=0., cptr=None):
```

Comment:
Or set `dt=None` as default to `Particle.__init__`?

Reply:
Indeed, this is what I did (Actually, I put it to nan, but None makes more sense. I'll check if it is possible)